mac.sh

```diff
@@ -242,66 +194,23 @@ install_postgresql
 brew_install_or_upgrade 'redis'
 brew_launchctl_restart 'redis'
 brew_install_or_upgrade 'sops'
-brew_install_or_upgrade 'the_silver_searcher'
-brew_install_or_upgrade 'vim'
-brew_install_or_upgrade 'ctags'
-brew_install_or_upgrade 'tmux'
-brew_install_or_upgrade 'reattach-to-user-namespace'
-brew_install_or_upgrade 'imagemagick'
 brew_install_or_upgrade 'n'
 sudo n stable
 brew_install_or_upgrade 'yarn'
 brew_install_or_upgrade 'rbenv'
 brew_install_or_upgrade 'ruby-build'
-brew_install_or_upgrade 'wget'
-brew_install_or_upgrade 'autojump'
 brew_install_or_upgrade 'openssl'
 brew unlink openssl && brew link openssl --force
-brew_install_or_upgrade 'z'
 
 brew_tap 'homebrew/cask'
-cask_install_or_upgrade 'chromedriver'
```

Comment:
is this an opinionated install? i thought this was required for running certain specs

Reply:
yeah sadly this is required if you want to run the feature spec suite locally :/

Reply:
ah yeah, for some reason I thought we removed most / all of that but will add back in

Reply:
someday @mttrms, someday </3